viam-cartographer/src/carto_facade/carto_facade_test.cc

```diff
@@ -616,6 +616,28 @@ BOOST_AUTO_TEST_CASE(CartoFacade_demo) {
                    VIAM_CARTO_POINTCLOUD_MAP_EMPTY);
     }
 
+    // GetInternalState
+    // Every call to AddSensorReading after the first one (for some
+    // unknown reason) should be persisted in the the internal state.
```

Comment:
[optional] This comment is similar to the one below. I think we could have just the comment below, where it's most relevant.

Reply:
done